src/braket/ir/jaqcd/__init__.py

```diff
@@ -18,16 +18,21 @@
     XY,
     YY,
     ZZ,
+    Amplitude_Damping,
```

Comment:
You should probably use the same naming convention as with the other gates (e.g. AmplitudeDamping instead of Amplitude_Damping).